substrafl/algorithms/pytorch/torch_base_algo.py

```diff
@@ -158,11 +158,12 @@ def _local_predict(self, predict_dataset: torch.utils.data.Dataset, predictions_
 
         self._model.eval()
 
-        predictions = torch.Tensor([]).to(self._device)
+        predictions = []
         with torch.inference_mode():
             for x in predict_loader:
                 x = x.to(self._device)
-                predictions = torch.cat((predictions, self._model(x)), 0)
+                predictions.append(self._model(x))
```

Comment:
Toying with the idea of making it more Pythonic with 
```
with torch.inference_mode():
    predictions = [self._model(x.to(self._device)) for x in predict_loader]
```
But not sure about readability 🤔 
WDYT @ThibaultFy ?

Reply:
Looks prettier for sure :)

Reply:
Let's go for pretty and efficient then :)

Reply:
I'm sorry but it's not clear to me what you want in the end: do you want me to modify with your 1-liner @SdgJlbl ?

Reply:
We usually let the person who open the PR to update it and merge it but as you prefer, we can take it from there too if you want !

Reply:
no no I can modify, just tell me clearly what I need to do

Reply:
Fixed!

Reply:
Perfect thanks a lot ! 😄

Reply:
I renamed the PR to match our usage. You can merge it when the test terminates 😄